test/e2e/integration/frontend-test/party-selection.ts

```diff
@@ -40,27 +151,77 @@ describe('Party selection', () => {
     cy.get(appFrontend.reporteeSelection.reportee).should('have.length', 1).contains('DDG');
   });
 
+  it('Org number should not be displayed for persons', () => {
+    mockResponses({
+      allowedToInstantiate: [ExamplePerson1, ExamplePerson2],
+      partyTypesAllowed: {
+        person: true,
+        subUnit: false,
+        bankruptcyEstate: false,
+        organisation: false,
+      },
+    });
+
+    cy.startAppInstance(appFrontend.apps.frontendTest);
+
+    // TODO: Implement the rest
```

Comment:
I guess this test can be deleted if not implemented, though, I see that it would have been failing if implemented (as mentioned from what I saw in the Percy screenshot).